pages/dkp/konvoy/2.1/backup/index.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: layout.pug
+navigationTitle: Back up and Restore
+title: Back up and restore
+menuWeight: 210
+excerpt: Back up and restore clusters
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
```

Comment:
I would recommend applying any of the relevant changes from the Kommander review above to this Konvoy version of the file, as well.

Reply:
I think this entire page should be deleted.   Konvoy no long provides Velero, only Kommander does.  I think it is confusing to have Kommander only things mentioned in the Konvoy docs.

If we think we need a 'backup' page in Konvoy, the contents should say something like "Ask of DKP 2.x, backup functionality has been moved to Kommander, see [here][link to kommander backup page]

Reply:
That's basically what we did in Kommander previously, too.